tests/e2e-test/templates/e2e-test-pod.yaml

```diff
@@ -2,13 +2,34 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: marinera-e2e-{{ randAlphaNum 5 | lower }}
-  annotations:
-    "helm.sh/hook": pre-install
-    "helm.sh/hook-delete-policy": hook-succeeded
```

Comment:
The test doesn't run without this part because the helm chart gets uninstalled before the pod starts. The other option is to use `--wait` when installing, but I couldn't make it work in branch `feature-test-sec-5` since the test has this error:
```
75s         Warning   FailedMount              pod/marinera-e2e-5gztu                                     MountVolume.SetUp failed for volume "kc-config" : object "feature-test-sec-5"/"marinera-e2e-keycloak-cm" not registered
```

Reply:
You're right. That explains why the tests stopped to fail. They did not execute anymore...;) Fixed it now with the hook weight